catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/ShareableMetacardPreIngestPluginTest.java

```diff
@@ -66,25 +66,25 @@ protected Subject getSubject() {
   }
 
   @Test
-  public void testCreateShouldIgnoreNonWorkspaceMetacards() throws Exception {
-    WorkspacePreIngestPlugin wpip = makePlugin(ImmutableMap.of());
+  public void testCreateShouldIgnoreNonShareableMetacards() throws Exception {
+    ShareableMetacardPreIngestPlugin wpip = makePlugin(ImmutableMap.of());
```

Comment:
✏️ looks like the variable name also needs an update in some of these methods

Reply:
Naming convention should be good in recent commit.